src/job-exit-spec/config/job-exit-spec.yaml

```diff
@@ -973,6 +973,24 @@ spec:
     - "Contact Cluster Admin"
 
 
+################################
+# Range: [-7499, -7400]
+# Owner: PAI_LAUNCHER WebServer
+# Description: Task Failure
+################################
+- code: -7400
+  phrase: TASK_STOPPED_ON_APP_COMPLETION
+  issuer: PAI_LAUNCHER
+  causer: USER_STOP
+  type: USER_STOP
```

Comment:
@sunqinzheng you may need to use code: -7400 or type: USER_STOP to judge whether the TaskState is Stop/Success/Failure...